apps/native/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@pierrelstan there file is generated by expo right? If this is the case, let's remove it from GitHub, we just keep minimum necessary file on codebase

Reply:
Yes, it's generated by Expo . I'll remove it.